### PR TITLE
Expand menu

### PR DESCRIPTION
**Pull request by ilcardella**
Hi, first you did a great job! I needed something like this and it works perfectly.
I have refactored a lot, so I do not expect you to like it as is, but maybe we can get something nice out of it.
Main changes:
- Created the menu to connect to specific countries.
- Created a menu to view the current connection status
- Created (empty at the moment) settings menu
- Refactored indicator update pattern. The Indicator class is in control of it.
- Removed notifications as the client app already pushes them

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -1,41 +1,58 @@
...
+install_indicator()
+{
+    # Installing indicator in opt directory
+    echo Installing Ubuntu NordVPN Indicator
+    sudo mkdir -p /opt/ubuntu-nordvpn-indicator/
+    sudo cp code/nordvpn_* /opt/ubuntu-nordvpn-indicator/
+
+    # Installing autostart desktop file
+    echo Making sure the indicator starts at boot using autostart
+    cp ubuntu-nordvpn-indicator.desktop ~/.config/autostart/
```
Comment (yorickvanzweeden) [inline install.sh]: If autostart does not exist (i.e.: Ubuntu 18.04), this willl crash
```suggestion
    mkdir -p ~/.config/autostart/
    cp ubuntu-nordvpn-indicator.desktop ~/.config/autostart/
```

File: code/nordvpn_indicator.py [hunk 2]
```diff
@@ -93,17 +98,54 @@ def build_menu(self):
...
+        menu_status.append(self.status_label)
+
+        menu.append(item_status)
+
+        # Create a submenu for the settings
+        menu_settings = gtk.Menu()
+        item_settings = gtk.MenuItem('Settings')
+        item_settings.set_submenu(menu_settings)
+        # TODO add settings
+        menu.append(item_settings)
```
Comment (yorickvanzweeden) [inline code/nordvpn_indicator.py]: Maybe it's better not to include this? As Settings does nothing yet

File: install.sh [hunk 3]
```diff
@@ -1,41 +1,58 @@
...
-
-# Installing indicator in opt directory
-echo Installing Ubuntu NordVPN Indicator
-sudo mkdir /opt/ubuntu-nordvpn-indicator/
-sudo cp code/nordvpn_* /opt/ubuntu-nordvpn-indicator/
-
-# Installing autostart desktop file
-echo Making sure the indicator starts at boot using autostart
-cp ubuntu-nordvpn-indicator.desktop ~/.config/autostart/
+install_client()
```
Comment (yorickvanzweeden) [inline install.sh:3]: I like this format!

File: install.sh [hunk 4]
```diff
@@ -1,41 +1,58 @@
...
+    # Prompting user to login
+    echo Logging into NordVPN
+    nordvpn login
+}
+
+install_deps()
+{
+    # Install gir1.2-appindicator
+    echo Installing AppIndicator
+    sudo apt-get install -y gir1.2-appindicator
```
Comment (yorickvanzweeden) [inline install.sh]: Python-gi includes module `gi`
```suggestion
    sudo apt-get install -y gir1.2-appindicator python-gi
```

File: code/nordvpn_indicator.py [hunk 5]
```diff
@@ -216,19 +289,34 @@ def status_check(self, _):
...
+        return self.connected
+
+    def get_countries(self):
+        """
+        Returns a list of string representing the available countries
+        """
+        countries = self.run_command('nordvpn countries')
+        if countries is None:
+            return []
+        return ''.join(countries).split(' ')[2].split()
```
Comment (yorickvanzweeden) [inline code/nordvpn_indicator.py]: This returns a sorted list of countries
```suggestion
        countries = countries.split(' ')[2].split()
        countries.sort()
        return countries
```

**Review by yorickvanzweeden — CHANGES_REQUESTED** [review]
Hey ilcardella, thank you for your additions! Please review my suggested changes

File: code/nordvpn_indicator.py (hunk 2 above)
Comment (ilcardella) [reply code/nordvpn_indicator.py]: Yeah sorry!

**Comment by ilcardella** [comment]
Hi, I have applied the changes for the review.
Anyway well done for the code. I just needed something like this!

**Review by yorickvanzweeden — APPROVED** [review]

**Pull request merged**